projects/kit/components/filter/test/filter.component.spec.ts

```diff
@@ -21,7 +21,7 @@ class ItemWithBadge {
     }
 
     public valueOf(): number | null {
-        return this.badgeValue !== undefined ? this.badgeValue : null;
+        return this.badgeValue ?? null;
```

Comment:
This is not an equivalent replacement, if `this.badgeValue` is `null`. It's okay for us?

Reply:
if `this.badgeValue` is `null` then 

`null !== undefined ? null : null;` -> `null`